orderer/common/multichannel/chainsupport.go

```diff
@@ -149,6 +150,23 @@ func (cs *ChainSupport) ProposeConfigUpdate(configtx *cb.Envelope) (*cb.ConfigEn
 		return nil, err
 	}
 
+	isOldV3 := cs.ChannelConfig().Capabilities().ConsensusTypeBFT()
+	isNewV3 := bundle.ChannelConfig().Capabilities().ConsensusTypeBFT()
+
+	// if we move from V2 to V3 we need the check the org endpoints, if one of them is empty return an error
+	if !isOldV3 && isNewV3 {
+		orderer, err := bundle.OrdererConfig()
+		if err {
+			return nil, errors.New("new config is missing orderer group")
+		}
+
+		for _, org := range orderer.Organizations() {
+			if len(org.Endpoints()) == 0 {
+				return nil, errors.Errorf("illegal orderer config detected during consensus metadata validation: endpoints of org %s are missing", org.Name())
```

Comment:
I don't see this covered in any test, am I wrong?

Reply:
Correct. I'll add some.

Reply:
And... this piece of code is not in the right place. It should be in `func (b *Bundle) ValidateNew(nb Resources) error `